Readme.md

```diff
@@ -0,0 +1,55 @@
+**The Readme is to detail the behaviour expected from the Stochiometry widget**
+
+1. When initializing the records of the experiment
+2. When changing the values of the experiment
+
+**Initializing the records of the experiment:**
+
+On launching the Stochiometry widget, to add to the page, all rows, except for row 1
+will be disabled for entry.
+
+For row 1 initial entry, the following are the permitted ways - 
+
+|  User Enters  | Automatically Calculated |
+| ------------- | ------------- |
+| amt and fw  | moles  |
```

Comment:
If we are distinguishing this from "fw and amt" we should probably change "and" to "then"